packages/web/src/components/gridView.tsx

```diff
@@ -76,6 +76,7 @@ export function GridView<T>(model: GridViewProps<T>) {
       <div className='grid-view-header'>
         {model.columns.map((column, i) => {
           return <div
+            key={model.columnTitle(column)}
```

Comment:
Just making sure, this is rendered only for the column header so no duplicate keys should end up here.